src/Basque/Mews.Fiscalizations.Basque/TicketBaiClient.cs

```diff
@@ -39,46 +39,25 @@ public async Task<SendInvoiceResponse> SendInvoiceAsync(SendInvoiceRequest reque
                 encoding: ServiceInfo.Encoding,
                 namespaces: NonEmptyEnumerable.Create(new XmlNamespace("http://www.w3.org/2000/09/xmldsig#"))
             ));
-            SignXml(xmlDoc.OwnerDocument);
+            TicketBaiSignature.Sign(xmlDoc.OwnerDocument, Certificate, Region);
```

Comment:
Are we happy that the tests cover the new stuff that is now happening in this method?

Reply:
Sure, we can have a ticket for writing some tests to verify the data in the XML.

Reply:
Why a ticket? A ticket just means we don't have to do it now. But why not do it now? Why should we code something and deploy it without having the tests first?